trailmap/acquisition_paths.py

```diff
@@ -0,0 +1,54 @@
+import networkx as nx
+from pprint import pprint
+import trailmap.pathway_analysis as pa
+
+
+def conduct_apa(facility_dict_in, facility_dict_out):
+    ''' '''
+    G = build_graph(facility_dict_in, facility_dict_out)
+
+    sources = list(node for node, in_deg in G.in_degree() if in_deg == 0)
+    sinks = list(node for node, out_deg in G.out_degree() if out_deg == 0)
+
+    print("\nSimple paths")
+    pathways = find_simple_paths(G, sources, sinks)
+    print(pathways)
+
+    return G, pathways
+
+
+def build_graph(facility_dict_in, facility_dict_out):
+    '''Builds NetworkX graph from Cyclus input file
+    '''
+    G = nx.DiGraph()
+    G.add_nodes_from(facility_dict_in.keys())
+    for reciever in facility_dict_in:
+        for incommod in facility_dict_in[reciever]:
```

Comment:
```suggestion
    for reciever, incommods in facility_dict_in.items():
        for incommod in incommods:
```

Reply:
This does not work as intended, trying to figure out why

Reply:
Figured it out, incorporated your comment. Thanks!